tests/FakerDotNet.Tests/Fakers/SpaceFakerTests.cs

```diff
@@ -0,0 +1,175 @@
+                .Returns(Data.StarClusters.First());
+
+            Assert.AreEqual(Data.StarClusters.First(), _spaceFaker.StarCluster());
+        }
+
+        [Test]
+        public void Constellation_returns_a_constellation()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(Data.Constellations)))
+                .Returns(Data.Constellations.First());
+
+            Assert.AreEqual(Data.Constellations.First(), _spaceFaker.Constellation());
+        }
+
+        [Test]
+        public void Star_returns_a_star()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(Data.Stars)))
+                .Returns(Data.Stars.First());
+
+            Assert.AreEqual(Data.Stars.First(), _spaceFaker.Star());
+        }
+
+        [Test]
+        public void Agency_returns_an_agency()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(Data.Agencies)))
+                .Returns(Data.Agencies.First());
+
+            Assert.AreEqual(Data.Agencies.First(), _spaceFaker.Agency());
+        }
+
+        [Test]
+        public void Agencyabv_returns_an_agencyabv()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(Data.AgencyAbvs)))
+                .Returns(Data.AgencyAbvs.First());
+
+            Assert.AreEqual(Data.AgencyAbvs.First(), _spaceFaker.AgencyAbv());
+        }
+
+        [Test]
+        public void NasaSpaceCraft_returns_a_nasaspacecraft()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(Data.NasaSpaceCraft)))
+                .Returns(Data.NasaSpaceCraft.First());
+
+            Assert.AreEqual(Data.NasaSpaceCraft.First(), _spaceFaker.NasaSpaceCraft());
+        }
+
+        [Test]
+        public void Company_returns_a_company()
+        {
+            A.CallTo(() => _fakerContainer.Random.Element(
+                    A<IEnumerable<string>>.That.IsSameSequenceAs(Data.Companies)))
+                .Returns(Data.Companies.First());
+
+            Assert.AreEqual(Data.Companies.First(), _spaceFaker.Company());
+        }
+
+        [Test]
+        public void DistanceMeasurement_returns_a_distancemeasurement_with_two_digit_number()
+        {
+            100.Times(() =>
+            {
+                A.CallTo(() => _fakerContainer.Random.Element(
+                        A<IEnumerable<string>>.That.IsSameSequenceAs(Data.DistanceMeasurements)))
+                    .Returns(Data.DistanceMeasurements.First());
+
+                var result = _spaceFaker.DistanceMeasurement();
+
+                Assert.That(Regex.IsMatch(result, $@"[1-9]\d {Data.DistanceMeasurements.First()}"), $"Result did not match: {result}");
```

Comment:
You can mock the call to the number faker so then you don't need to perform a regex :+1: